### ISSUE TITLE
Support Gromacs

### ISSUE THREAD

**Issue by jhprinz** [markovmodel/adaptivemd#8]
This is interesting for @nuria. Since she already has script to run gromacs we could support these in `adaptivemd`. That should be a quick addition.

**Comment by nsplattner** [comment]
I think this should be feasible. But I think for both, GROMACS and Acemd the important thing is not so much to have everything available, but just to document where a runnable script for a given MD engine can be added and what functionality it needs to provide. We can do this for Gromacs as an example.

**Comment by jhprinz** [comment]
Good point. I think it would be a little like openmm. My implementation covers most common cases (as long as you create the .xml files) I would like to have something similar (mostly transfer the openmm code) for AceMD and Gromacs. In the process we should document how to do and adapt it to your liking.
